### PR TITLE
feat(cutlass): add support for custom timestamp columns

### PR DESCRIPTION
**Pull request by M0dEx**
This PR adds support for custom timestamp columns accepting both microseconds since UNIX epoch as a long and ISO 8601 formatted UTC timestamp.

### CODE REVIEW

**Comment by M0dEx** [comment]
Hi @jerrinot!

Just letting you know I closed the previous PR and opened this one to add the timestamp functionality to the new API.

File: core/src/main/java/io/questdb/cutlass/line/AbstractLineSender.java [hunk 1]
```diff
@@ -154,6 +156,27 @@ public final AbstractLineSender boolColumn(CharSequence name, boolean value) {
         return field(name, value);
     }
 
+    public AbstractLineSender fieldTimestamp(CharSequence name, long value) {
```
Comment (jerrinot) [inline core/src/main/java/io/questdb/cutlass/line/AbstractLineSender.java:159]: this is unnecessary. 

I'm keeping the old `field()` methods only for the sake of backward compatibility. 
my thinking is that after a few releases - when users hopefully migrate to the new `Sender` interface - we would remove the old methods entirely.

File: core/src/main/java/io/questdb/cutlass/line/AbstractLineSender.java [hunk 2]
```diff
@@ -154,6 +156,27 @@ public final AbstractLineSender boolColumn(CharSequence name, boolean value) {
...
+        return this;
+    }
+
+    @Override
+    public final AbstractLineSender timestampColumn(CharSequence name, long value) {
+        return fieldTimestamp(name, value);
+    }
+
+    @Override
+    public final AbstractLineSender timestampColumn(CharSequence name, CharSequence value) throws LineSenderException {
```
[the remark this answers is not in the document]
Comment (M0dEx) [reply core/src/main/java/io/questdb/cutlass/line/AbstractLineSender.java:167]: Oh, alright.

I can confirm there would be interest from out company right away, but I understand the need for preventing API creep.

**Comment by jerrinot** [comment]
Looks pretty good now, thank you!

**Pull request merged**